test/testdata/deployednettemplates/recipes/alphanet/genesis.json

```diff
@@ -1,59 +1,159 @@
 {
-	"NetworkName": "alphanet",
+	"NetworkName": "",
 	"VersionModifier": "",
-	"ConsensusProtocol": "alpha1",
```

Comment:
why did we change genesis'proto?

Reply:
I believe we update this for performance testing. The live network genesis doesn't change (it's in the installer dir) but we need to keep this updated.

Reply:
This is just to make the Jenkins tests running the "alphanet" recipe start up from the latest version — the real alphanet genesis is in installer/genesis/alphanet/genesis.json. I guess another approach would be to have an `alphalatest` or `alphafuture` version defined to set this to